docs/guides/deployment/bare_metal.rst

```diff
@@ -77,6 +77,10 @@ default. You can start the server by enabling the unit.
 This will start the server on port 5656, and the data directory will be
 ``/var/lib/edgedb/1/data``.
 
+.. warning::
+
+    ``edgedb-server`` cannot be run as root.
+
```

Comment:
@fmoor Had to push again. Brain is permanently stuck on Markdown mode. 😅 

Placement look good here?

Reply:
looks fine to me.